### ISSUE TITLE
Jak and Daxter Collection no longer boots since #3820

### ISSUE THREAD

**Issue by PTKickass** [RPCS3/rpcs3#3828]
Since #3820, the Jak and Daxter Collection (BCUS98281) refuses to boot. Can confirm it worked in the previous build (#3806)

Here's what happens when you try to run the game on #3820 and newer builds (using both SPU and PPU precise interpreters and the OpenGL renderer):

![captura de ecra 565](https://user-images.githubusercontent.com/20132880/33516537-485dee66-d76c-11e7-9a19-f2959cb73828.png)

Sometimes it doesn't crash, and the console presents a "Access violation writing location 0x200" or "Access violation writing location 0x8" error message.
It also crashes using any kind of configuration (Recompilers, Vulkan, etc.)

Log File: [RPCS3.log](https://github.com/RPCS3/rpcs3/files/1523872/RPCS3.log)

System Specs:
CPU - Intel Core i5-4690k
GPU - Nvidia GeForce GTX 970
4GB of DDR3 RAM

**Comment by snowboardkids** [comment]
Same issue here.. please update

**Comment by rohit-n** [comment]
Can confirm this issue as well.

**Comment by isJuhn** [comment]
The game works again if you revert commit `cd9070e60da825bd042046628e4c23c7d747bbf4` aka
```
mfc: Fix support for list transfer stall bit with partial support for…
… out of order execution - Also give Sync commands a size so they are properly detected by queue checks
```
In #3820 
![rpcs3_2018-01-13_20-51-02](https://user-images.githubusercontent.com/26828095/35151650-6e8de392-fd1f-11e7-9b11-8139a58b74ab.png)

**Comment by elad335** [comment]
Fixed with #4088
